ntbs-service/Models/Entities/TreatmentEvent.cs

```diff
@@ -57,5 +58,12 @@ public class TreatmentEvent : ModelBase, IHasRootEntity
 
         string IHasRootEntity.RootEntityType => RootEntities.Notification;
         string IHasRootEntity.RootId => NotificationId.ToString();
+
+        public IList<TreatmentEventType> EditableTreatmentEventTypes => new List<TreatmentEventType>
+        {
+            Enums.TreatmentEventType.TreatmentOutcome,
+            Enums.TreatmentEventType.TreatmentRestart
+        };
```

Comment:
I think we can make this static, rather than having each entity hold its own copy of the list?